docs/public/_redirects

```diff
@@ -196,6 +196,18 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /zh/api/time-picker/ /zh/x/api/date-pickers/time-picker/ 301
 /pt/api/time-picker/ /pt/x/api/date-pickers/time-picker/ 301
 
+### Group & Pivot page split https://github.com/mui/mui-x/pull/4441
+/x/react-data-grid/group-pivot#tree-data /x/react-data-grid/tree-data 301
```

Comment:
URI fragment is never sent to the server it won't do anything 

```suggestion
/x/react-data-grid/group-pivot /x/react-data-grid/tree-data 301
```

Reply:
Ahhh, right!

Reply:
That's a bummer, we'll have to use client-side redirect then.
We'll encounter the same problem when splitting Columns page https://github.com/mui/mui-x/pull/4600